.github/workflows/sync-generated-files.yml

```diff
@@ -0,0 +1,38 @@
+name: Sync generated files
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          version: 10.x
```

Comment:
I _believe_ this needs to be `node-version` now https://github.com/actions/setup-node#usage

Reply:
Should be good now!